src/graphql/mirror.js

```diff
@@ -318,6 +318,18 @@ export class Mirror {
           CREATE INDEX idx_connection_entries__connection_id
           ON connection_entries (connection_id)
         `,
+        dedent`\
+          CREATE TABLE network_log (
+              rowid INTEGER PRIMARY KEY,
+              query TEXT NOT NULL,
+              query_parameters TEXT NOT NULL,
+              request_timestamp INTEGER NOT NULL,
```

Comment:
Optional, but consider making these nullable for symmetry? There’s no
critical application logic that relies on them being required.

Reply:
updated